samples/dnn/dasiamrpn_tracker.cpp

```diff
@@ -0,0 +1,460 @@
+//DaSiamRPN tracker.
+//Original paper: https://arxiv.org/abs/1808.06048
+//Link to original repo: https://github.com/foolwood/DaSiamRPN
+//Links to onnx models:
+//network:     https://www.dropbox.com/s/rr1lk9355vzolqv/dasiamrpn_model.onnx?dl=0
+//kernel_r1:   https://www.dropbox.com/s/999cqx5zrfi7w4p/dasiamrpn_kernel_r1.onnx?dl=0
+//kernel_cls1: https://www.dropbox.com/s/qvmtszx5h339a0w/dasiamrpn_kernel_cls1.onnx?dl=0
+
+#include <iostream>
+#include <cmath>
+
+#include <opencv2/dnn.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+
+using namespace cv;
+using namespace cv::dnn;
+
+const char *keys =
+        "{ help     h   | | Print help message. }"
+        "{ input    i   | | Full path to input video folder. (empty for camera) }"
+        "{ net          |./dasiamrpn_model.onnx | Full path to onnx model of net.}"
+        "{ kernel_cls1  |./dasiamrpn_kernel_cls1.onnx | Full path to onnx model of net.}"
+        "{ kernel_r1    |./dasiamrpn_kernel_r1.onnx | Full path to onnx model of net.}";
+
+typedef struct Box
+{
+    float x;
+    float y;
+    float w;
+    float h;
+} Box;
```

Comment:
Please reuse `cv::Rect2f`

Reply:
Thanks for reviewing my code. I have used the Rect2f in the new version.